### PR TITLE
Update the specs to match the current implementation

### PR DESCRIPTION
**Pull request by micbar**
# Description

- Cleanup and remove unneeded attributes
- Change the routes to match the official specs

### CODE REVIEW

**Review by butonic — APPROVED** [review]
nice!

File: api/openapi-spec/v0.0.yaml [hunk 1]
```diff
@@ -213,11 +187,29 @@ paths:
         default:
           $ref: '#/components/responses/error'
       x-ms-docs-operation-type: operation
+    post:
+      tags:
+        - drives
+      summary: Create a new navigation property of a specific drive
```
Comment (refs) [inline api/openapi-spec/v0.0.yaml:193]: The operation id doesn't match the description. I think it is a copy paste leftover from the previous operation.

**Review by refs — COMMENTED** [review]
Can't request changes because it is merged, but worth having a look at that description @micbar ^^

File: api/openapi-spec/v0.0.yaml (hunk 1 above)
Comment (micbar) [reply api/openapi-spec/v0.0.yaml:193]: operation-id is `drives.CreateDrive`

**Pull request merged**